apiary.apib

```diff
@@ -1,80 +1,138 @@
 FORMAT: 1A
-HOST: http://polls.apiblueprint.org/
+HOST: https://example.org
 
 # Cosmos API Stake Capital
 
-Polls is a simple API allowing consumers to view polls and vote in them.
+API allowing clients to view state and retrieve data from the network.
 
-## Questions Collection [/questions]
+# GET /status
 
-### List All Questions [GET]
+Get Tendermint status including node info, pubkey, latest block hash, app hash, block height and time.
+
+From Tendermint RPC endpoint:
+```bash
+curl 'localhost:26657/status'
+```
 
 + Response 200 (application/json)
 
-        [
-            {
-                "question": "Favourite programming language?",
-                "published_at": "2015-08-05T08:40:51.620Z",
-                "choices": [
-                    {
-                        "choice": "Swift",
-                        "votes": 2048
-                    }, {
-                        "choice": "Python",
-                        "votes": 1024
-                    }, {
-                        "choice": "Objective-C",
-                        "votes": 512
-                    }, {
-                        "choice": "Ruby",
-                        "votes": 256
+        {
+            "jsonrpc": "2.0",
+            "id": "",
+            "result": {
+            
+            
+                "node_info": {
+                        "protocol_version": {
+                            "p2p": "4",
+                            "block": "7",
+                            "app": "0"
+                        },
+                        "id": "53729852020041b956e86685e24394e0bee4373f",
+                        "listen_addr": "10.0.2.15:26656",
+                        "network": "test-chain-Y1OHx6",
+                        "version": "0.24.0-2ce1abc2",
+                        "channels": "4020212223303800",
+                        "moniker": "ubuntu-xenial",
+                        "other": {
+                            "tx_index": "on",
+                            "rpc_addr": "tcp://0.0.0.0:26657"
+                        }
+                    },
+                    "sync_info": {
+                        "latest_block_hash": "F51538DA498299F4C57AC8162AAFA0254CE08286",
```

Comment:
we need this: 
```
 "latest_block_hash": "F51538DA498299F4C57AC8162AAFA0254CE08286",
                        "latest_app_hash": "0000000000000000",
                        "latest_block_height": "18",
                        "latest_block_time": "2018-09-17T11:42:19.149920551Z"
```

Reply:
The Stake Capital API will return clean data to save $ : 
we have endpoint setup: 
"network": "game_of_stakes_3",
"version": "0.27.3"

and another endpoint status (called more frequently): 
      "latest_block_hash": "C3A6C2B4298D6A81A55132AA78AF1D159EE1D276052855B30D77B41837FD24CE",
      "latest_app_hash": "02E672666712B0B234A4F144F04F52A5E4A5C56FC248A54344084279121D7EE4",
      "latest_block_height": "224796",
      "latest_block_time": "2019-01-22T15:02:25.142458104Z",

Reply:
comments taken into account in latest commit.